spec/swagger_schemas/requests/application_accept_request.rb

```diff
@@ -29,3 +29,42 @@
     },
   },
 }.freeze
+
```

Comment:
Following the comment above.. I'd suggest to do something like this so we don't add `APPLICATION_ACCEPT_REQUEST_V3` to api v1 & v2 yaml files.

```
APPLICATION_ACCEPT_REQUEST = {
  v1: {
    description: "A NPQ application acceptance request",
    type: :object,
    properties: {
      data: {
        description: "A NPQ application acceptance request data",
        type: :object,
        required: %i[type attributes],
        properties: {
          type: {
            description: "The data typed",
            type: :string,
            example: "npq-application-accept",
          },
          attributes: {
            description: "A NPQ application acceptance request attributes",
            type: :object,
            properties: {
              funded_place: {
                description: "This field indicates whether the application is funded",
                nullable: false,
                type: :boolean,
                required: false,
                example: true,
              },
            },
          },
        },
      },
    },
  },
}.tap { |h|
  h[:v2] = h[:v1]
  h[:v3] = {
    description: "A NPQ application acceptance request",
    type: :object,
    properties: {
      data: {
        description: "A NPQ application acceptance request data",
        type: :object,
        required: %i[type attributes],
        properties: {
          type: {
            description: "The data typed",
            type: :string,
            example: "npq-application-accept",
          },
          attributes: {
            description: "A NPQ application acceptance request attributes",
            type: :object,
            properties: {
              funded_place: {
                description: "This field indicates whether the application is funded",
                nullable: false,
                type: :boolean,
                required: false,
                example: true,
              },
              schedule_identifier: {
                description: "The new schedule of the participant",
                nullable: false,
                type: :string,
                example: "npq-leadership-spring",
                required: false,
                enum: %w[
                  npq-aso-march
                  npq-aso-june
                  npq-aso-november
                  npq-aso-december
                  npq-ehco-march
                  npq-ehco-june
                  npq-ehco-november
                  npq-ehco-december
                  npq-leadership-autumn
                  npq-leadership-spring
                  npq-specialist-autumn
                  npq-specialist-spring
                ],
              },
            },
          },
        },
      },
    },
  }
}.freeze
```

Reply:
Thanks Leandro 👍